BetweenTwoSets.cs

```diff
@@ -0,0 +1,60 @@
+using System.CodeDom.Compiler;
+using System.Collections.Generic;
+using System.Collections;
+using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Serialization;
+using System.Text.RegularExpressions;
+using System.Text;
+using System;
+
+class Result
+{
+
+    /*
+     * Complete the 'getTotalX' function below.
+     *
+     * The function is expected to return an INTEGER.
+     * The function accepts following parameters:
+     *  1. INTEGER_ARRAY a
+     *  2. INTEGER_ARRAY b
+     */
+    public static int getTotalX(List<int> a, List<int> b)
+    {
+        var count=0;
+        for(int num=a.Max();num<=b.Min();++num){
```

Comment:
When you call Max()method does it compute the max value every time? If yes its is better to call it only once and store it another variable

Reply:
When you call Min()method does it compute the min value every time? If yes its is better to call it only once and store it another variable

Reply:
@narayanagunturi yes the Min()method will be called for all the iterations in the for loop so it will be costly process.Will change the code by using another variable to store.

Reply:
@narayanagunturi changes done